src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1432,4 +1432,70 @@ void TwitchMessageBuilder::deletionMessage(const DeleteAction &action,
     builder->message().timeoutUser = "msg:" + action.messageId;
 }
 
+void TwitchMessageBuilder::modsOrVipsSystemMessage(QString prefix,
+                                                   QStringList users,
+                                                   TwitchChannel *channel,
+                                                   MessageBuilder *builder)
+{
+    builder->emplace<TimestampElement>();
+    builder->message().flags.set(MessageFlag::System);
+    builder->message().flags.set(MessageFlag::DoNotTriggerNotification);
+    builder->emplace<TextElement>(prefix, MessageElementFlag::Text,
+                                  MessageColor::System);
+    bool hasColor = false;
+    bool isFirst = true;
+    MessageColor color;
+    for (const QString &username : users)
+    {
+        if (!isFirst)
+        {
+            // this is used to add the ", " after each but the last entry
+            builder->emplace<TextElement>(", ", MessageElementFlag::Text,
```

Comment:
```suggestion
            builder->emplace<TextElement>(",", MessageElementFlag::Text,
```

Before:
![image](https://user-images.githubusercontent.com/41973452/130380162-353d93ab-ac65-475d-b362-4032bc7d8443.png)

After:
![image](https://user-images.githubusercontent.com/41973452/130380063-f560784e-badf-4449-aea3-592ed4238010.png)